### PR TITLE
Cleanup boilerplate code

### PR DESCRIPTION
**Pull request by josephmfaulkner**
One Application, ready for action

### CODE REVIEW

**Comment by josephmfaulkner** [comment]
@feydan : There we go, one shiny pull request ready for review.

File: backend-nodejs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Node build artifacts
+node_modules
```
Comment (goonchersen) [inline backend-nodejs/.gitignore:2]: I typically like to include a trailing slash on directories whose contents I want ignored.  It's just a visual cue for me to understand "node_modules is a _directory_, and this won't apply to files named "node_modules"

File: backend-nodejs/.sequelizerc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```
Comment (goonchersen) [inline backend-nodejs/.sequelizerc:1]: It's probably helpful to eliminate all things _sequelize_ from the pull request.  I doubt that the Trash API app will require any persistence (beyond committing images to an S3 bucket).  All of the _sequelize_ artifacts will just continue to clutter things up.

File: backend-nodejs/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+[Main](../README.md)
+# Webapp Boilerplate (backend)
+This repository goes with [webapp-boilerplate-frontend](https://github.com/josephmfaulkner/webapp-boilerplate-frontend)
```
Comment (goonchersen) [inline backend-nodejs/README.md:3]: You'll likely want to eliminate all references to your own GitHub account and repositories

File: backend-nodejs/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+
+## Deploying to Heroku
+Instructions for deploying this app to Heroku can be found [by clicking here](./infrastructure/README.md)
+
+
+## Local DB Development
```
Comment (goonchersen) [inline backend-nodejs/README.md:10]: Remove

File: backend-nodejs/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+docker-compose up
+```
+In another terminal
+```
+npx sequelize-cli db:migrate
+npx sequelize-cli db:seed:all
+```
+**Open PG Admin:** [localhost:8080](localhost:8080), **PW:** password
+
+**Add DB to PG Admin:** Servers > Create > Server
```
Comment (goonchersen) [inline backend-nodejs/README.md:21]: Ibid.

File: backend-nodejs/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+version: "3.9"
```
Comment (goonchersen) [inline backend-nodejs/docker-compose.yml:1]: Again, persistence probably isn't necessary, so I wouldn't bother having the PostgreSQL database spun up here

**Review by goonchersen — CHANGES_REQUESTED** [review]
I began working through this PR file-by-file, but then decided that a single comment associated with the review is probably all that's necessary.  Suggestions are as follows:

- Eliminate persistence capabilities and artifacts.  At this point, with no applicable user story, persistence isn't necessary, and only clutters the code.  One of the tenets of lean software development is to implement only what is necessary, and I think that that tenet is applicable here.

- The use of _passport.js_ is great, but I'm not certain that authentication is a valid user story.  We don't have the sort of application that requires authentication -- _yet_, at least -- so I'd take it out.  I _can_ imagine the need for authorization using OAuth and a valid JWT token in request headers, but on the surface that's sufficient to POST images to any data set.  Again, thinking lean, I'd remove the authentication work.

- I believe that the only known use case for the backend up to this point is a simple endpoint protected by a valid JWT token where a user can upload a photograph, leverage the image recognition model in association with an object detection filter, and return the annotated photograph to the user by way of an HTTP response.  The team collectively has questions to Win pending, and I believe it would be helpful to wait until we have greater certainty and not risk implementing what won't be necessary.

- Keep the _express.js_ server configuration there.  I see a need for just a single endpoint that returns a 200, with the possibility of protecting the endpoint by means of the JWT token.

**Pull request merged**